### PR TITLE
Make schema validation part of dryrun

### PR DESCRIPTION
**Pull request by relud**
schema validation reintroduced dry running all queries CI, this fixes that by moving schema validation into the `dry-run-sql` CI step.

### CODE REVIEW

File: bigquery_etl/schema/__init__.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-from typing import Any, Dict, Optional, List
+from typing import Any, Dict, List, Optional
 
 import attr
-import os
 import yaml
+from google.cloud import bigquery
 
-from bigquery_etl.dryrun import DryRun
+from .. import dryrun
```
Comment (relud) [inline bigquery_etl/schema/__init__.py:13]: this allows `bigquery_etl/dryrun.py` to do `from .schema import SCHEMA_FILE, Schema` without a circular import exception.

File: bigquery_etl/dryrun.py [hunk 2]
```diff
@@ -446,47 +545,3 @@ def find_next_word(target, source):
...
-        result = p.map(sql_file_valid, sorted(sql_files), chunksize=1)
-    if all(result):
-        exitcode = 0
-    else:
-        exitcode = 1
-    sys.exit(exitcode)
-
-
-if __name__ == "__main__":
-    main()
```
Comment (relud) [inline bigquery_etl/dryrun.py:492]: removed in favor of `bqetl dryrun`

File: bigquery_etl/cli/dryrun.py [hunk 3]
```diff
@@ -42,61 +45,60 @@
...
-
-                try:
-                    client.query(query, job_config=job_config)
-                    click.echo(f"{sqlfile:59} OK")
-                    return True
-                except Exception as e:
-                    click.echo(f"{sqlfile:59} ERROR: {e}")
-                    return False
-
-        sql_file_valid = gcp_dryrun
```
Comment (relud) [inline bigquery_etl/cli/dryrun.py:99]: moved to `DryRun.dry_run_result` to make `--use-cloud-function=False` compatible with `--validate-schemas`

File: bigquery_etl/cli/dryrun.py [hunk 4]
```diff
@@ -27,8 +30,8 @@
     """,
 )
 @click.argument(
-    "path",
-    default="sql/",
+    "paths",
+    nargs=-1,
```
Comment (relud) [inline bigquery_etl/cli/dryrun.py:34]: replaced `path` with `paths` so that `bqetl dryrun` could replace `python3 -m bigquery_etl.dryrun`

**Review by scholtzan — APPROVED** [review]

**Pull request merged**